File: beginner/1048.py
salary = float(input())
PERCENTUAL15 = 15
PERCENTUAL12 = 12
PERCENTUAL10 = 10
PERCENTUAL7 = 7
PERCENTUAL4 = 4
percentualutilized = 0
moneyEarned = 0

if salary >= 0 and salary <= 400.00:
	moneyEarned = (salary*PERCENTUAL15) / 100
	percentualutilized = PERCENTUAL15
	
if salary >= 400.01 and salary <= 800.00:
	moneyEarned = (salary*PERCENTUAL12) / 100
	percentualutilized = PERCENTUAL12

if salary >= 800.01 and salary <= 1200.00:
	moneyEarned = (salary*PERCENTUAL10) / 100
	percentualutilized = PERCENTUAL10
	
if salary >= 1200.01 and salary <= 2000.00:
	moneyEarned = (salary*PERCENTUAL7) / 100
	percentualutilized = PERCENTUAL7

if salary > 2000.00:
	moneyEarned = (salary*PERCENTUAL4) / 100
	percentualutilized = PERCENTUAL4

print(f"Novo salario: {(salary+moneyEarned):.2f}")
print(f"Reajuste ganho: {moneyEarned:,.2f}")
print(f"Em percentual: {percentualutilized} %")

File: beginner/1048.c
# include <stdio.h>
# define PERCENTUAL15 15
# define PERCENTUAL12 12
# define PERCENTUAL10 10
# define PERCENTUAL7 7
# define PERCENTUAL4 4

int main(){
	float salary = 0.0, moneyEarned = 0;
	
	scanf("%f", &salary);
	int percentualUtilized = 0;
	
	if( salary >= 0 && salary <= 400.00 ){
		moneyEarned = (salary*PERCENTUAL15) / 100;
		percentualUtilized = PERCENTUAL15;
	}
	
	if( salary >= 400.01 && salary <= 800.00 ){
		moneyEarned = (salary*PERCENTUAL12) / 100;
		percentualUtilized = PERCENTUAL12;
	}
	
	if( salary >= 800.01 && salary <= 1200.00 ){
		moneyEarned = (salary*PERCENTUAL10) / 100;
		percentualUtilized = PERCENTUAL10;
	}
	
	if( salary >= 1200.01 && salary <= 2000.00 ){
		moneyEarned = (salary*PERCENTUAL7) / 100;
		percentualUtilized = PERCENTUAL7;
	}
	
	if( salary > 2000.00 ){
		moneyEarned = (salary*PERCENTUAL4) / 100;
		percentualUtilized = PERCENTUAL4;
	}
	
	printf("Novo salario: %.2f\n", salary + moneyEarned );
	printf("Reajuste ganho: %.2f\n", moneyEarned );
	printf("Em percentual: %i %%\n", percentualUtilized );
	
	return 0;
}
